go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/metal3-io/baremetal-operator
 
-go 1.21
+go 1.22
```

Comment:
This is not necessary unless a dependency is forcing us to.

Reply:
Sure, I see the concern of unnecessary changes on stable branches, should be fixed now!